data-visualize.qmd

```diff
@@ -732,7 +728,7 @@ ggplot(penguins, aes(x = flipper_length_mm, y = body_mass_g)) +
 
 ### Three or more variables
 
-One way to add additional variables to a plot is by mapping them to an aesthetic.
+As we saw in our Ultimate Goal, we can incorporate more variables into a plot by mapping them to additional aesthetics.
```

Comment:
You used "ultimate goal" in other places

Reply:
It is referring to that plot, but maybe not necessary to do so.